scripts/phylum-init.sh

```diff
@@ -68,6 +68,21 @@ require_command() {
     fi
 }
 
+usage() {
+    cat 1>&2 <<EOF
+phylum-init.sh [options]
+
+Fetch and install the phylum CLI.
+
+Options
+    -t, --target       Specify the target triple to install
+    -y, --yes          Do not prompt for comfirmation during install
+    --no-verify        Disable signature verification (NOT RECOMMENDED)
+    -h, --help         Show this help message
+
```

Comment:
This is extremely nitpicky, but generally I'd avoid unnecessary newlines at the end of a command. Would be perfectly fine with this, but there's no need to take up more space than necessary.

The standard is generally *not* to include a newline. At least that's what any command I could come up with does.

Reply:
Simple enough and that makes sense

Reply:
it was changed...resolving...